flow/types.js

```diff
@@ -86,27 +90,33 @@ declare module 'icarus-backend' {
     data: any
   }
 
-  declare type TxInput = {
-    txDbId: number,
+  declare type TxInputDbResult = {
+    txDbId: string,
     address: string,
     value: number,
     hash: string,
     index: number,
     isGenesis?: boolean,
+    txOutDbId: string,
```

Comment:
why do we need `txOutDbId` for tx inputs?

Reply:
Everything that wants to have tokens appended with the current logic, needs txOutDbId